auxiliary_tools/metrics_checker.py

```diff
@@ -0,0 +1,60 @@
+#!usr/bin/env python
+from __future__ import division, print_function
+
+import argparse
+import os
+
+"""
+Usage: metrics_checker.py [options]
+Example: python metrics_checker.py -t /lcrc/group/e3sm/public_html/e3sm_diags_test_data/unit_test_complete_run/expected/previous_output/all_sets_v2_6_1_20220328_d62f554/ -r /lcrc/group/e3sm/public_html/e3sm_diags_test_data/unit_test_complete_run/expected/all_sets/
```

Comment:
This example works (albeit with a few `Failed to open file` errors). 

However, I ran the tests and then `python metrics_checker.py -t ../tests/integration/all_sets_results_test/ -r ../tests/integration/integration_test_images/` which isn't that helpful because `viewer` doesn't have a `table-data` subdirectory. What needs to happen for that subdirectory to exist?

Reply:
I guess this is expected, the results under both path here mentioned are not a typical e3sm_diags run directory.